pysumma/preprocessing/create_initial_condition.py

```diff
@@ -0,0 +1,152 @@
+from osgeo import gdal
+import pandas as pd
+import os
+import numpy as np
+import json
+import pkg_resources
+from .create_local_attribute import LocalAttributes
+
+PARAMETER = pkg_resources.resource_filename(
+        __name__, 'meta/hru_ic.json')
+with open(PARAMETER, 'r') as f:
+    hru_ic = json.load(f)
+
+PARAMETER = pkg_resources.resource_filename(
+        __name__, 'meta/mid_ic.json')
+with open(PARAMETER, 'r') as f:
+    mid_ic = json.load(f)
+
+ssurgo_soil_db = pkg_resources.resource_filename(
+        __name__, 'meta/ssurgo_soils_db.csv')
+
+class InitialCondition(object):
```

Comment:
I think we need some documentation in here, especially for the more complex functions.

Reply:
Also, could you explain exactly what assumptions/datasets we're using to create this here?